### DISCUSSION TITLE
Is it still possible to use it?

### DISCUSSION THREAD

**Discussion by MrObvi** [doville/rn-fitness-tracker#139] [Q&A] [answered]
I tried installing via: 
npm install @kilohealth/rn-fitness-tracker
and I got error:
npm error Cannot read properties of null (reading 'children')

**Comment by lukebars** [comment, marked as answer]
Hello,

Yes, of course it is possible. We are using it in production. 

Could you provide a clean repository with an example of your integration?

**Reply by MrObvi** [reply]
Thanks for the response I managed to install it using yarn.
